Last-Update: 2019-09-09
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@debian.org>
Description: switch ostree-go to packaged fork - original library is not in Debian.

--- a/vendor/github.com/containers/image/ostree/ostree_dest.go
+++ b/vendor/github.com/containers/image/ostree/ostree_dest.go
@@ -25,9 +25,9 @@
 	"github.com/containers/storage/pkg/archive"
 	"github.com/klauspost/pgzip"
 	"github.com/opencontainers/go-digest"
 	selinux "github.com/opencontainers/selinux/go-selinux"
-	"github.com/ostreedev/ostree-go/pkg/otbuiltin"
+	"github.com/sjoerdsimons/ostree-go/pkg/otbuiltin"
 	"github.com/pkg/errors"
 	"github.com/vbatts/tar-split/tar/asm"
 	"github.com/vbatts/tar-split/tar/storage"
 )
--- a/vendor/github.com/containers/image/ostree/ostree_src.go
+++ b/vendor/github.com/containers/image/ostree/ostree_src.go
@@ -17,9 +17,9 @@
 	"github.com/containers/image/types"
 	"github.com/containers/storage/pkg/ioutils"
 	"github.com/klauspost/pgzip"
 	digest "github.com/opencontainers/go-digest"
-	glib "github.com/ostreedev/ostree-go/pkg/glibobject"
+	glib "github.com/sjoerdsimons/ostree-go/pkg/glibobject"
 	"github.com/pkg/errors"
 	"github.com/vbatts/tar-split/tar/asm"
 	"github.com/vbatts/tar-split/tar/storage"
 )
--- a/vendor/github.com/containers/storage/pkg/ostree/ostree.go
+++ b/vendor/github.com/containers/storage/pkg/ostree/ostree.go
@@ -13,10 +13,10 @@
 	"unsafe"
 
 	"github.com/containers/storage/pkg/idtools"
 	"github.com/containers/storage/pkg/system"
-	glib "github.com/ostreedev/ostree-go/pkg/glibobject"
-	"github.com/ostreedev/ostree-go/pkg/otbuiltin"
+	glib "github.com/sjoerdsimons/ostree-go/pkg/glibobject"
+	"github.com/sjoerdsimons/ostree-go/pkg/otbuiltin"
 	"github.com/pkg/errors"
 )
 
 // #cgo pkg-config: glib-2.0 gobject-2.0 ostree-1
